### PR TITLE
Describe current test naming conventions in the contributing guidelines

### PR DESCRIPTION
**Pull request by rumyantseva**
# Description

Closes FerretDB/engineering#66.

It's a part of the main issue, based on the outcome of #2816. 
This PR only describes the current guidelines. The new guidelines will be added in separate PRs together with the changes to meet these new guidelines. 

## Readiness checklist

- [ ] I added/updated unit tests.
- [ ] I added/updated integration/compatibility tests.
- [ ] I added/updated comments and checked rendering.
- [x] I made spot refactorings.
- [ ] I updated user documentation.
- [ ] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

File: integration/indexes_compat_test.go [hunk 1]
```diff
@@ -287,8 +287,8 @@ func TestIndexesCompatCreate(t *testing.T) {
 	}
 }
 
-// TestIndexesCreateRunCommand tests specific behavior for index creation that can be only provided through RunCommand.
-func TestIndexesCompatCreateRunCommand(t *testing.T) {
+// TestIndexesCreateCommandCompat tests specific behavior for index creation that can be only provided through RunCommand.
```
Comment (w84thesun) [inline integration/indexes_compat_test.go:290]: Should it be `TestCreateIndexesCommandCompat`?  😀

**Review by w84thesun — COMMENTED** [review]
Nice to have those guidelines!

File: integration/indexes_compat_test.go (hunk 1 above)
Comment (rumyantseva) [reply integration/indexes_compat_test.go:290]: true!

**Review by noisersup — DISMISSED** [review]
Makes sense! No changes needed except the one above.

**Review by noisersup — DISMISSED** [review]
LGTM!

**Review by w84thesun — DISMISSED** [review]

File: integration/indexes_compat_test.go [hunk 2]
```diff
@@ -559,7 +559,7 @@ func TestIndexesCompatDrop(t *testing.T) {
 	}
 }
 
-func TestIndexesCompatDropRunCommand(t *testing.T) {
+func TestIndexesDropCommandCompat(t *testing.T) {
```
Comment (chilagrow) [inline integration/indexes_compat_test.go:562]: Okay so it's `TestDropIndexesCommandCompat` now according to the point 4.

Comment (rumyantseva) [reply integration/indexes_compat_test.go]: yes! thank you!

**Review by chilagrow — APPROVED** [review]
:rocket:

**Review by w84thesun — APPROVED** [review]

**Review by AlekSi — APPROVED** [review]

**Pull request merged**